spec/rails_helper.rb

```diff
@@ -95,6 +95,13 @@
   config.before do
     # Worker jobs shouldn't linger around between tests
     Sidekiq::Worker.clear_all
+
+    # NOTE: @citizen428 needed while we delegate from User to Profile to keep
+    # spec changes limited for the time being.
+    ProfileField.destroy_all
```

Comment:
This data refresh is going to slow down our specs considerably. Does every spec start failing without this? Is there any way we can limit this to a smaller subset of specs?

Reply:
> Does every spec start failing without this?

No, but too many to fix right now in one go. 

> Is there any way we can limit this to a smaller subset of specs?

That's how I started, by making a shared context. When I realized how many specs I'd need to add this too I went for this instead. Let me try and just move this to the `before(:suite)` block so we only create them once.